### PR TITLE
fixes #775: Update deps in docker-server to match global deps

### LINKED ISSUE #775: lhci-server docker image crashes when passed storage env variables in Node14
**Describe the bug**
Docker Image for 0.9.0 crashes when passed ENV variables for Database parameters. This is due to Node14 and an older version of Sequelize in the server package. Upgrading Sequelize resolved the issue for me. 

**To Reproduce**
1. Pull latest image down `patrickhulce/lhci-server:0.9.0`
2. Run a local postgres DB - `docker run --name postgres-docker -e POSTGRES_PASSWORD=postgres -p 5432:5432 -d postgres`
3. Run `docker run -p 9001:9001  -e LHCI_STORAGE__SQL_DIALECT=postgres -e LHCI_STORAGE__SQL_CONNECTION_URL=postgres://postgres:postgres@host.docker.internal:5432/postgres patrickhulce/lhci-server:0.9.0` * if not on mac change host.docker.internal to reference the postgres container or host address.
4. Docker crashes
5. Go to step 3 but with release 0.8.2 and it works as expected.

**Expected behavior**
Application runs and does not crash as seen in 0.8.2

**Logs/Screenshots**
If applicable, add LHCI logs or screenshots to help explain your problem.

### issue when running with docker

```
➜  docker run -p 9001:9001  -e LHCI_STORAGE__SQL_DIALECT=postgres -e LHCI_STORAGE__SQL_CONNECTION_URL=postgres://postgres:postgres@host.docker.internal:5432/postgres lhci-server
npm info it worked if it ends with ok
npm verb cli [
npm verb cli   '/usr/local/bin/node',
npm verb cli   '/usr/local/bin/npm',
npm verb cli   'start',
npm verb cli   '--verbose',
npm verb cli   '-ddd'
npm verb cli ]
npm info using npm@6.14.16
npm info using node@v14.19.1
npm verb run-script [ 'prestart', 'start', 'poststart' ]
npm info lifecycle lhci@0.0.0~prestart: lhci@0.0.0
npm info lifecycle lhci@0.0.0~start: lhci@0.0.0

> lhci@0.0.0 start /usr/src/lhci
> lhci server --config=./lighthouserc.json

npm verb lifecycle lhci@0.0.0~start: unsafe-perm in lifecycle true
npm verb lifecycle lhci@0.0.0~start: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/usr/src/lhci/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
npm verb lifecycle lhci@0.0.0~start: CWD: /usr/src/lhci
npm sill lifecycle lhci@0.0.0~start: Args: [ '-c', 'lhci server --config=./lighthouserc.json' ]
npm sill lifecycle lhci@0.0.0~start: Returned: code: 0  signal: null
npm info lifecycle lhci@0.0.0~poststart: lhci@0.0.0
npm verb exit [ 0, true ]
npm timing npm Completed in 1613ms
npm info ok
```

### on Local machine from docs/recipes/docker-server directory:
```
➜ LHCI_STORAGE__SQL_DIALECT="postgres" LHCI_STORAGE__SQL_CONNECTION_URL="postgres://postgres:postgres@localhost:5432/postgres" npm --verbose start
npm info it worked if it ends with ok
npm verb cli [
npm verb cli   '/Users/user1/.nvm/versions/node/v14.19.0/bin/node',
npm verb cli   '/Users/user1/.nvm/versions/node/v14.19.0/bin/npm',
npm verb cli   '--verbose',
npm verb cli   'start'
npm verb cli ]
npm info using npm@6.14.16
npm info using node@v14.19.0
npm verb run-script [ 'prestart', 'start', 'poststart' ]
npm info lifecycle lhci@0.0.0~prestart: lhci@0.0.0
npm info lifecycle lhci@0.0.0~start: lhci@0.0.0

> lhci@0.0.0 start /Users/user1/Codebase/forks/lighthouse-ci/docs/recipes/docker-server
> lhci server --config=./lighthouserc.json

npm verb lifecycle lhci@0.0.0~start: unsafe-perm in lifecycle true
npm verb lifecycle lhci@0.0.0~start: PATH: /Users/user1/.nvm/versions/node/v14.19.0/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/user1/Codebase/forks/lighthouse-ci/docs/recipes/docker-server/node_modules/.bin:/Users/user1/.nvm/versions/node/v14.19.0/bin:/usr/local/opt/openssl@1.1/bin:/usr/local/sbin:/usr/local/opt/openjdk@11/bin:/Users/user1/.nvm/versions/node/v12.14.1/bin:/Users/user1/.nvm/versions/node/v12.14.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/usr/local/opt/openssl@1.1/bin:/usr/local/sbin:/usr/local/opt/openjdk@11/bin:/Users/user1/.nvm/versions/node/v12.14.1/bin
npm verb lifecycle lhci@0.0.0~start: CWD: /Users/user1/Codebase/forks/lighthouse-ci/docs/recipes/docker-server
npm info lifecycle lhci@0.0.0~poststart: lhci@0.0.0
npm verb exit [ 0, true ]
npm timing npm Completed in 1154ms
npm info ok 

```

**Environment (please complete the following information):**
 - OS: MacOS Host within Docker environment
 - Browser: N/A
 - Version: 0.9.0

**Additional context**
Seems to work locally when using Node12.

### PR DESCRIPTION
**Pull request by metalix2**
Main fix here is to set the same version the docker server dependencies use as the main packages.

I've tested this and it seems to fix the #775 issue.

### CODE REVIEW

File: docs/recipes/docker-server/package.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
     "@lhci/cli": "0.9.0",
     "@lhci/server": "0.9.0",
     "mysql2": "^2.1.0",
-    "pg": "^7.12.1",
+    "pg": "^8.2.1",
```
Comment (metalix2) [inline docs/recipes/docker-server/package.json:11]: https://github.com/GoogleChrome/lighthouse-ci/blob/9deb96d0070bfbde2711547693bc98591e8c544b/package.json#L83

File: docs/recipes/docker-server/package.json [hunk 2]
```diff
@@ -8,8 +8,8 @@
...
     "pg-hstore": "^2.3.3",
-    "sqlite3": "^4.0.6"
+    "sqlite3": "^5.0.0"
```
Comment (metalix2) [inline docs/recipes/docker-server/package.json:13]: https://github.com/GoogleChrome/lighthouse-ci/blob/9deb96d0070bfbde2711547693bc98591e8c544b/package.json#L83

File: docs/recipes/docker-server/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM node:14-buster-slim
+FROM node:16-buster-slim
```
Comment (metalix2) [inline docs/recipes/docker-server/Dockerfile:1]: With node 14 in maintenance I feel moving to 16 is idea as it's in Active development still.

**Comment by metalix2** [comment]
Hey @patrickhulce @connorjclark,

PR Ready for review - with the end of support for Node 12 ending in less than 2 weeks (30th of April). It would be good to release a fix to resolve #775 🙏 

Look forward to your response.

File: docs/recipes/docker-server/Dockerfile [hunk 4]
```diff
@@ -1,4 +1,4 @@
-FROM node:14-buster-slim
+FROM node:14-bullseye-slim
```
[the remark this answers is not in the document]
Comment (metalix2) [reply docs/recipes/docker-server/Dockerfile:1]: https://github.com/GoogleChrome/lighthouse-ci/runs/6060006286?check_suite_focus=true#step:11:1885

**Review by patrickhulce — APPROVED** [review]
thanks @metalix2 !

**Comment by metalix2** [comment]
Hey @patrickhulce any news on when this could be merge and released? 🙏

**Comment by patrickhulce** [comment]
@connorjclark any chance you have bandwidth for release?

**Pull request merged**